### ISSUE TITLE
🟡 P2 — Investigate `ml_enhanced_*` systematic -2.00% stop-loss pattern

### ISSUE THREAD

**Issue by eltonaguiar** [eltonaguiar/findtorontoevents_antigravity.ca-archive-2026-05-23#690]
## Problem

Multiple `ml_enhanced_<SYMBOL>` strategies show suspiciously uniform -2.00% average PnL with 0% win rate, suggesting they are all hitting an identical hard stop-loss.

### Affected strategies (7d window, CRYPTO)

| Strategy | n | PF | WR | Avg PnL |
|---|---|---|---|---|
| `ml_enhanced_WLDUSDT` | 4 | 0.00 | 0.0% | -2.00% |
| `ml_enhanced_APEUSDT` | 4 | 4.50 | 75.0% | +1.75% |
| `ml_enhanced_TIAUSDT` | 4 | 1.50 | 50.0% | +0.50% |
| `ml_enhanced_AAVEUSDT` | 3 | 0.00 | 0.0% | -2.00% |
| `ml_enhanced_NEARUSDT` | 2 | 0.00 | 0.0% | -2.00% |
| `ml_enhanced_ADAUSDT` | 2 | 0.00 | 0.0% | -2.00% |
| `ml_enhanced_AVAXUSDT` | 2 | 0.00 | 0.0% | -2.00% |
| `ml_enhanced_ETHUSDT` | 2 | 0.00 | 0.0% | -2.00% |
| `ml_enhanced_SOLUSDT` | 2 | 0.00 | 0.0% | -2.00% |
| `ml_enhanced_LINKUSDT` | 2 | 0.00 | 0.0% | -2.00% |
| `ml_enhanced_BTCUSDT` | 1 | 0.00 | 0.0% | -2.00% |
| `ml_enhanced_OPUSDT` | 1 | 0.00 | 0.0% | -2.00% |
| `ml_enhanced_DOGEUSDT` | 1 | ∞ | 100% | +3.00% |

**Pattern:** 11 out of 13 `ml_enhanced_*` symbols show exactly -2.00% when they lose. This is not a natural distribution.

## Hypotheses

1. **Hard stop at -2%.** The ML model may be calibrated with a fixed -2% stop-loss. If the model's direction prediction is wrong, every pick hits the same stop.
2. **Timeframe mismatch.** Model trained on 1h/4h data but stop calibrated for 15m volatility = guaranteed stop-out on noise.
3. **Adverse selection in symbol list.** The specific symbols with `ml_enhanced_*` wrappers may be the ones the base model performs worst on (hence why they got individual wrappers).

## Recommendation

1. **Verify SL calibration.** Check if all `ml_enhanced_*` picks use `stop_loss = entry_price * 0.98`.
2. **Audit model confidence vs outcome.** Is there an inverse correlation? (High model confidence = worse outcome = overfitting.)
3. **If hypothesis 1 confirmed:** Either widen the stop to -3% or -4%, or reduce position size so that -2% stops don't dominate P&L.
4. **Symbol-level audit:** Some symbols (APE, TIA, DOGE) show positive results. Consider dropping the consistently losing symbols from the ML wrapper list.

## Related
- PR #668 (Copilot added `ml_gatekeeper` feature flag — not yet reviewed)

**Comment by eltonaguiar** [comment]
**Data check 2026-05-17:** The systematic -2.00% stop-loss pattern no longer appears in current `closed_picks.json`.

```
Total ml_enhanced picks: 778
Picks at exactly -2.00% pnl_pct: 0 (0.0%)
PnL stats: mean=-0.020%, median=+0.002%
```

The pattern was present when this issue was filed (n=4 per symbol, uniform -2.00%). Current data shows 778 ml_enhanced picks with no concentration at -2.00%. Either the SL calibration was adjusted or the affected picks resolved through a different path.

Note: ML_ENHANCED symbol-level blocks are active for BTCUSDT (stack variant) and LONG direction is preferred for lightgbm variant per mutation analysis in `quality_gates.py:4100-4107`. The `quan_engine` matrix symbol gates also block many underperforming CRYPTO symbols.

_Verified 2026-05-17 by Claude Code automated session review. Close this issue if the pattern is confirmed resolved._
